cypress/component/NodeAdmonition.cy.tsx

```diff
@@ -0,0 +1,51 @@
+import NodeAdmonition from '../../src/components/NodeAdmonition';
+
+describe('<NodeAdmonition />', () => {
+  it('renders admonitions for provided nodes', () => {
+    cy.mount(<NodeAdmonition nodes={['OpenNeuro', 'EBRAINS']} onDismiss={cy.stub()} />);
+
+    cy.get('[data-cy="openneuro-alert"]').should('exist');
+    cy.contains('The OpenNeuro node is being actively annotated').should('be.visible');
+
+    cy.get('[data-cy="ebrains-alert"]').should('exist');
+    cy.contains('The EBRAINS node is being actively annotated').should('be.visible');
+  });
```

Comment:
Please take a look at and update the test, I believe it's still expecting and asserting over a stacked version rather the current tab implementation.